File: 241022/숫자의 계절은/season-of-num.cpp
#include <iostream>
using namespace std;

int main() {
    int month;
    cin >> month;

    if (month >= 12 || month <= 2) {
        cout << "Winter";
    } else if (month <= 5) {
        cout << "Spring";
    } else if (month <= 8) {
        cout << "Summer";
    } else {
        cout << "Fall";
    }
    return 0;
}

File: 241022/숫자의 계절은/season-of-num.js
const fs = require("fs");
let month = Number(fs.readFileSync(0).toString().trim());

if (3 <= month && month <= 5) {
    console.log("Spring");
} else if (6 <= month && month <= 8) {
    console.log("Summer");
} else if (9 <= month && month <= 11) {
    console.log("Fall");
} else {
    console.log("Winter");
}